### PR TITLE
Py311 fix

### PR DESCRIPTION
**Pull request by duckduckdoof**
These changes are updates to the resource.path() changes in python3.9.

### CODE REVIEW

File: awe_lexica/setup/data.py [hunk 1]
```diff
@@ -753,8 +753,13 @@ def loadViewpointLexicon(viewpointLex='argVoc.csv'):
     """
     stancePerspectiveVoc = {}
 
-    with resources.path('awe_lexica.data',
-                         viewpointLex) as filepath:
+    # This method for resources is deprecated; switching to recommended calls
+    # with resources.path('awe_lexica.data',
+    #                      viewpointLex) as filepath:
```
Comment (bradley-erickson) [inline awe_lexica/setup/data.py]: I don't think this comment is necessary.

File: awe_lexica/setup/data.py [hunk 2]
```diff
@@ -826,8 +831,11 @@ def to_disk():
         filename = "{variable_name}.json".format(
             variable_name=export
         )
-        with resources.path('awe_lexica.json_data',
-                            filename) as outputfile:
+        # with resources.path('awe_lexica.json_data',
+        #                     filename) as outputfile:
```
Comment (bradley-erickson) [inline awe_lexica/setup/data.py]: Commented code should be removed unless it is crucial for understanding what the current code does. 

As a dev, I don't care that it used to use resources.path

File: setup.cfg [hunk 3]
```diff
@@ -28,7 +28,7 @@ classifiers =
 include_package_data = True
 python_requires = >=3.9
 install_requires =
-  spacy>=3.1.0,<3.4.0
+  spacy>=3.0.0,<3.6.0
```
Comment (bradley-erickson) [inline setup.cfg]: Why are upper bounding the Spacy version? Spacy is already on 3.7 according to PyPi.

**Review by bradley-erickson — CHANGES_REQUESTED** [review]
See individual comments.

File: setup.cfg (hunk 3 above)
Comment (duckduckdoof) [reply setup.cfg]: I will remove this -- I think it was originally there when I was experimenting with 3.12 (and associated issues with spacy/textblob/etc.)

File: awe_lexica/setup/data.py [hunk 4]
```diff
@@ -903,8 +905,11 @@ def levenshtein_ratio_and_distance(s, t, ratio_calc=False):
 
 def processConcretes():
     rawconcretes = {}
-    with resources.path('awe_lexica.data',
-                        'Concr_Base.csv') as filepath:
+    # with resources.path('awe_lexica.data',
+    #                     'Concr_Base.csv') as filepath:
```
Comment (bradley-erickson) [inline awe_lexica/setup/data.py]: Remove unused comments (there is another chunk of code below with it as well)

File: setup.cfg [hunk 5]
```diff
@@ -28,7 +28,7 @@ classifiers =
 include_package_data = True
 python_requires = >=3.9
 install_requires =
-  spacy>=3.1.0,<3.4.0
+  spacy>=3.0.0
```
Comment (bradley-erickson) [inline setup.cfg]: Is changing from 3.1 to 3.0 going to cause any issues?

Comment (duckduckdoof) [reply setup.cfg]: from what I've tested on 3.9 and 3.11, no; although, I don't remember changing the version from 3.1 to 3.0. I'll change this to the original 3.1

**Review by bradley-erickson — APPROVED** [review]

**Comment by bradley-erickson** [comment]
It all looks good to me.

**Pull request merged**